common/changes/@typespec/rest/fixResourceLogic_2023-09-29-18-58.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@typespec/rest",
+      "comment": "Fix issue where getResourceTypeKey did not examine base classes.",
```

Comment:
I would word this in terms of the TypeSpec change, or at least add that context? E.g. "Fix: @key can now appear on the base model of a resource"

Reply:
Updated